packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -194,7 +194,12 @@ interface ShippingDefaultValues {
 }
 
 interface CustomerDefaultValues {
+    mode?: string;
```

Comment:
```suggestion
type Mode = 'billing' | 'shipping';

interface CustomerDefaultValues {
    mode?: Mode;
```

Reply:
We can also use enum here as well.

Reply:
Already created the suggested change, thank you for the comment.